### ISSUE TITLE
Closing brace dropped when uploading ill-styled code to gist

### ISSUE THREAD

**Issue by excsniper** [arduinodiscord/bot-legacy#16]
**To Reproduce**
User posted in discord:
```
void setup (){
  pinMode(8, OUTPUT);
  pinMode(2, INPUT);
  
}

void loop () {
  if (digitalRead(2) == HIGH){
  digitalWrite (8,HIGH);
  }

else {
  digitalWrite (8, LOW);
}
}
```

Gist:
https://gist.github.com/ArduinoDiscordBot/761bbf4dcc67908243a137181ba0acdb

**Comment by max-bromberg** [comment]
Will be looking into this. Sorry it's taken me so long :)

**Comment by max-bromberg** [comment]
This is fixed in faa2e5b11434792663b5d7fefe9a74abc1734a45
